translations/base-es.yaml

```diff
@@ -880,8 +880,8 @@ settings:
             title: Deshabilitar grilla
             description: Deshabilitar la grilla puede ayudar con el rendimiento. ¡También hace
                 que el juego se vea más limpio!
-        clearCursorOnDeleteWhilePlacing:
-            title: Limpiar el cursos al apretar click derecho
+        clearOnDeleteWhilePlacing:
```

Comment:
I think you can't change this key ... correct? If you do, the translation string will not be picked up.

Reply:
yeah, the key is not the problem, the problem is "title: Limpiar el *cursor* al apretar click derecho"

Reply:
why was the key changed, then?

Reply:
@neftalito I understand what you say. 

Let me explain:

You want to correct line L884 because `cursos` is wrong. You want to correct it to `cursor`, and your correction looks good. However, you can't change the key of this translation string (L883). 

If you do, the string will not get translated.

Reply:
Oh, yeah, you are right. Im sorry, didnt saw i changed the key value. My bad